AppTaskQueue/appscale/taskqueue/queue.py

```diff
@@ -125,6 +125,7 @@ def __init__(self, queue_info, app):
         self.task_retry_limit = retry_params['task_retry_limit']
 
     self.validate_config()
+    self.prepared_statements = {}
```

Comment:
How long does Queue object usually live? Couldn't this dict be cause of a memory leak? If it can live quite long, then it makes sense to limit size of the dict. As an option third party lib can be used (https://pypi.python.org/pypi/expiringdict/1.1.2) or we could implement our own.